bird_classify.py

```diff
@@ -88,7 +90,7 @@ def user_selections():
                         help='File path to deterrent sound')
     parser.add_argument('--print', default=False, required=False,
                         help='Print inference results to terminal')
-    parser.add_argument('--training', default=False, required=False,
+    parser.add_argument('--training', default='False', required=False, choices=['True', 'False'],
```

Comment:
Would `parser.add_argument('--training', action='store_true')` work better? Please check details here https://docs.python.org/3/library/argparse.html. In this case `args.training` would be a boolean.